<?xml version="1.0"?>

<launch>
<arg name="robot_name" default="pioneer"/>
<arg name="update_rate_info" value="1.0" />
      
<group ns="$(arg robot_name)">
  <node pkg="tf2_ros" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 map $(arg robot_name)/odom" />

  <!-- <node pkg="tuw_multi_robot_ctrl" type="tuw_velocity_controller" name="trajectory_controller" output="screen" > -->
      <!--remap from="cmd_vel" to="cmd_vel_not_connected"/-->
  <!-- </node> -->
  
  <!-- <node pkg="tuw_multi_robot_local_behavior_controller" type="mbf_controller_node" name="mbf_local_behavior" output="screen"> -->
  <node pkg="tuw_multi_robot_local_behavior_controller" type="tuw_local_behavior_controller_node" name="local_behavior" output="screen">
    <param name="robot_name" value="$(arg robot_name)" />
    <param name="robot_radius" value="0.3" />
    <remap from="robot_info" to="/robot_info" />
  </node>
  
  <node pkg="nodelet" type="nodelet" name="self_localization" args="standalone FakePoseEstimationNodelet"/>
  
</group>
  
</launch>
